src/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+interface AvatarProps {
```

Comment:
I think we do not want to use a interface here, since no one actually is going to extend the Avatar but rather use it as it is. Therefore, I'd recommend to convert the Avatarprops from an interface to a type.

Next to that, we always export our types also, so it would be better to also export it for the sake of consistency.

I have seen that in the mumble design, there is also an option to have an edit Icon on the avatar, but only in XL size. Is there a reason that we currently do not have it?

Reply:
@malinovic fixed
I also replaced some other interfaces in the code for consistency.